src/js/backup_restore.js

```diff
@@ -165,13 +170,72 @@ function configuration_backup(callback) {
                     configuration.GPS_CONFIG = jQuery.extend(true, {}, GPS_CONFIG);
                     configuration.COMPASS_CONFIG = jQuery.extend(true, {}, COMPASS_CONFIG);
                 }
+                if (semver.gte(CONFIG.apiVersion, "1.36.0")) {
+                    configuration.BEEPER_CONFIG = jQuery.extend(true, {}, BEEPER_CONFIG);
+                }
 
                 save();
             }
         }
+
+        if (GUI.configuration_loaded === true) {
+            return fetch_unique_data_item();
+        }
+        var promises = [ getAdvancedConfig(), getSensorConfig(), getCraftName(), getBoardAlignmentConfig(), getMixerConfig(), getBeeperConfig() ];
+        Promise.all(promises).
```

Comment:
This is a risky way of doing things. `Promise.all` does not specify if the promises are resolved in sequence or in parallel. Even if it works for now, the implementation might change (or might be different on a different platform), resulting in the MSP calls being executed in parallel.
These should be sequentialised like in:

https://github.com/smoriarty21/betaflight-configurator/blob/70c839eb3a3f869672af15992e883774d5497a00/src/js/tabs/pid_tuning.js#L23-L43